meta/send.rb

```diff
@@ -224,6 +224,22 @@
 end
 
 Mutant::Meta::Example.add do
+  source 'foo.values_at(a, b)'
```

Comment:
I think that we can reduce the verbosity of the meta DSL a bit, so that you do not have to re-specify "adjacent mutations" that have to be present to make the source of the "mutation under specification" valid.

I avoided doing this until its painful enough, IMO the "its painful enough" state is reached.